lib/IRGen/GenValueWitness.cpp

```diff
@@ -831,13 +832,22 @@ static void addValueWitness(IRGenModule &IGM,
 
   // Try to use a standard function.
   switch (index) {
-  case ValueWitness::Destroy:
+  case ValueWitness::Destroy: {
+    bool isClangType =
+        (concreteType.getStructOrBoundGenericStruct() &&
+         concreteType.getStructOrBoundGenericStruct()->getClangDecl());
+    // Emit ClangRecordTypeInfo::destroy in a function and add that function to
```

Comment:
Is this comment up to date? I can't find `ClangRecordTypeInfo`

Reply:
Nope, I need to change this to `AddressOnlyClangRecordTypeInfo`. Good catch.